src/fiatExchanges/quotes/FiatConnectQuote.test.ts

```diff
@@ -28,310 +28,362 @@ const mockExchangeRates = {
   cEUR: '2',
 }
 
+const getQuote = (baseFcQuote: FiatConnectQuoteSuccess, fiatAccountSchema: FiatAccountSchema) => {
+  const quoteCopy = _.cloneDeep(baseFcQuote)
+  quoteCopy.fiatAccount.BankAccount = {
+    ...quoteCopy.fiatAccount.BankAccount,
+    fiatAccountSchemas: [
+      {
+        fiatAccountSchema: fiatAccountSchema,
+        allowedValues: {
+          institutionName: ['Bank A', 'Bank B'],
+        },
+      },
+    ],
+  }
+  return quoteCopy
+}
+
+const baseFcQuote = mockFiatConnectQuotes[1] as FiatConnectQuoteSuccess
+
+const accountNumberQuote = getQuote(baseFcQuote, FiatAccountSchema.AccountNumber)
+const ibanQuote = getQuote(baseFcQuote, FiatAccountSchema.IBANNumber)
+
 describe('FiatConnectQuote', () => {
-  describe('constructor', () => {
-    it('throws an error if fiatAccountType is not supported', () => {
-      expect(
-        () =>
-          new FiatConnectQuote({
-            flow: CICOFlow.CashIn,
-            quote: mockFiatConnectQuotes[1] as FiatConnectQuoteSuccess,
-            fiatAccountType: 'Foo' as FiatAccountType,
-          })
-      ).toThrow()
-    })
-    it('throws an error if at least one fiatAccountSchema is not supported', () => {
-      const quoteData = {
-        ...mockFiatConnectQuotes[1],
-        fiatAccount: {
-          BankAccount: {
-            fiatAccountSchemas: [
-              {
-                fiatAccountSchema: 'SomethingUnsupported' as FiatAccountSchema,
-                allowedValues: {},
-              },
-            ],
+  describe.each([
```

Comment:
similar here comment here as FiatDetailsScreen.test.tsx, it should be sufficient to test just a single schema for most tests.